packages/grafana-ui/src/components/ColorPicker/ColorPicker.story.tsx

```diff
@@ -44,9 +49,7 @@ export const basic = () => {
   );
 };
 
-export const seriesColorPicker = () => {
-  const { enableNamedColors } = getColorPickerKnobs();
-
+export const seriesColorPicker: Story<ColorPickerProps> = ({ enableNamedColors }) => {
```

Comment:
```suggestion
export const SeriesColorPicker: Story<ColorPickerProps> = ({ enableNamedColors }) => {
```

Reply:
This gives off an error because SeriesColorPicker is already defined in this story. In scenarios like this, what do you advise we do? @Clarity-89

Reply:
ah, I see. In this case you can rename the story to `Series` or `SeriesPicker` since it's used in the context of ColorPicker (as a submenu).

Reply:
And I can't figure a better name for this story other than `seriesColorPicker`😝

Reply:
> ah, I see. In this case you can rename the story to `Series` or `SeriesPicker` since it's used in the context of ColorPicker (as a submenu).

Okay. This works too 👍